deployment/live/aws/test/README.md

```diff
@@ -0,0 +1,222 @@
+
+### With fake chains
+
+On the VM, run the following command to bring up the TesseraCT:
+
+```bash
+go run ./cmd/aws \
+  --http_endpoint=localhost:6962 \
+  --roots_pem_file=./internal/testdata/fake-ca.cert \
+  --origin=test-static-ct \
+  --bucket=${TESSERACT_BUCKET_NAME} \
+  --db_name=tesseract \
+  --db_host=${TESSERACT_DB_HOST} \
+  --db_port=3306 \
+  --db_user=tesseract \
+  --db_password=${TESSERACT_DB_PASSWORD} \
+  --dedup_path=test-static-ct
+```
+
+In a different terminal you can either mint and submit certificates manually, or
+use the [ct_hammer
+tool](https://github.com/google/certificate-transparency-go/blob/master/trillian/integration/ct_hammer/main.go)
+to do this.
+
+#### Generate chains manually
+
+Generate a chain manually. The password for the private key is `gently`:
+
+```bash
+mkdir -p /tmp/httpschain
+openssl genrsa -out /tmp/httpschain/cert.key 2048
+openssl req -new -key /tmp/httpschain/cert.key -out /tmp/httpschain/cert.csr -config=internal/testdata/fake-ca.cfg
+openssl x509 -req -days 3650 -in /tmp/httpschain/cert.csr -CAkey internal/testdata/fake-ca.privkey.pem -CA internal/testdata/fake-ca.cert -outform pem -out /tmp/httpschain/chain.pem -provider legacy -provider default
+cat internal/testdata/fake-ca.cert >> /tmp/httpschain/chain.pem
+```
+
+Finally, submit the chain to the TesseraCT:
+
+```bash
+go run github.com/google/certificate-transparency-go/client/ctclient@master upload --cert_chain=/tmp/httpschain/chain.pem --skip_https_verify --log_uri=http://localhost:6962/test-static-ct
+```
+
+#### Automatically generate chains
+
+Save the TesseraCT repo's path:
+
+```bash
+export TESSERACT_REPO=$(pwd)
+```
+
+Clone the [certificate-transparency-go](https://github.com/google/certificate-transparency-go) repo, and from there run:
+
+```bash
+go run ./trillian/integration/ct_hammer/ \
+  --ct_http_servers=localhost:6962/${TESSERA_BASE_NAME} \
+  --max_retry=2m \
+  --invalid_chance=0 \
+  --get_sth=0 \
+  --get_sth_consistency=0 \
+  --get_proof_by_hash=0 \
+  --get_entries=0 \
+  --get_roots=0 \
+  --get_entry_and_proof=0 \
+  --max_parallel_chains=4 \
+  --skip_https_verify=true \
+  --operations=10000 \
+  --rate_limit=150 \
+  --log_config=${TESSERACT_REPO}/testdata/hammer.cfg \
+  --testdata_dir=./trillian/testdata/
+```
+
+### With real HTTPS certificates
+
+We'll run a TesseraCT and copy certificates from an existing RFC6962 log to it.
+It uses the [ct_hammer tool from certificate-transparency-go](https://github.com/google/certificate-transparency-go/tree/aceb1d4481907b00c087020a3930c7bd691a0110/trillian/integration/ct_hammer).
+
+First, set a few environment variables:
+
+```bash
+export TESSERACT_REPO=$(pwd)
+export SRC_LOG_URI=https://ct.googleapis.com/logs/xenon2022
+```
+
+Then, get fetch the roots the source logs accepts, and edit configs accordingly.
+To do so, clone the [certificate-transparency-go](https://github.com/google/certificate-transparency-go) repo, and from there run:
```

Comment:
Makes sense to me to copy these instructions for now since thy are the same for GCP. Let's send another PR afterwards to run these without cloning the repo.